azure/kubernetes/aks-kubernetes.md

```diff
@@ -0,0 +1,211 @@
+# Service Approval Accelerator for Azure Kubernetes Service (AKS)
+
+## Identity & access management
```

Comment:
Identity & Access Management

We need to keep it consistent across the document.

Reply:
@TLATER - I have addressed in the following PR to your fork ...
https://github.com/TLATER/cloud-service-certification/pull/2